Python/errors.c

```diff
@@ -98,7 +98,6 @@ PyErr_SetObject(PyObject *exception, PyObject *value)
     exc_value = _PyErr_GetTopmostException(tstate)->exc_value;
     if (exc_value != NULL && exc_value != Py_None) {
         /* Implicit exception chaining */
-        Py_INCREF(exc_value);
```

Comment:
Not sure whether `exc_value` may somehow be freed as the result of calling arbitrary Python code below. Maybe it's safer to preserve this `Py_INCREF` and insert `Py_DECREF` into the right place instead?

Reply:
Yes, it  is safer to preserve this Py_INCREF.